public/static/js/init.py

```diff
@@ -28,21 +142,55 @@ def clear(self):
     def write(self, text):
         self.buffer += text
 
-
 debug_output = DebugOutput()
+debug_context= DebugContext()
 test_output = TestOutput()
 
 active_breakpoints = set()
 test_inputs = []
 test_outputs = []
 step_into = False
 
+# setup turtle library
+
+with open("turtle.py", "w") as file:
```

Comment:
a bit confused by this

Reply:
There may be a better way but to try to get PLS / python compatibility, by creating a turtle.py on the pyodide temporary session file system, it allows you to "import turtle" as part of your python example code and then create a turtle 
t = turtle.turtle().

A similar suggestion in the teams conversation regarding the tk stub to allow key press messages to be passed back to bouncing ball in a python compatible fashion.